rate-limiter/src/main/resources/Dockerfile

```
@@ -0,0 +1,48 @@
+# --------------------------------------------------------------------
+# Copyright (c) 2020, WSO2 LLC. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------
+
+FROM envoy-ratelimit-service:patched
```

?? Let's have a proper version here

---

This is the Rls build from the forked repo and Rls do not maintain version numbers. They use the commit ID instead.
We can have the branch name 'choreo' of the forked repo as the label as follows.

```suggestion
FROM envoy-ratelimit-service:choreo
```

wdyt?

---

Then let's keep the patched tag as it is.. We can move to the commit id with their own docker image.